### ISSUE TITLE
Priority 1 accessibility issue: Focus order on keyboard navigation

### ISSUE THREAD

**Issue by alicerichmond** [wellcomecollection/wellcomecollection.org#6548]
**What**

The text found under each image on this page, providing details on the exhibit, receives focus when navigating via keyboard, using the tab key; however, the element is not actionable. This may be disorientating for users navigating via the use of the keyboard alone as it can be difficult to track focus if it lands on unexpected objects within the page. Users may assume it to be focusable because it is interactive and then be confused as to why they cannot action the content. This also causes the need for additional key-presses when navigating the page, which can become strenuous for users with mobility impairments.

**Keyboard only comments:**

 “When testing this page, I found that under each of the photos there is what appears to be a link that gets highlighted, but when I try to click on it nothing happens. This is confusing as I would only expect elements I can interact with to receive focus.”

**Code ref:** #main > article > div.bg-cream > div:nth-child(1) > div > div > div:nth-child(2) > div > div > div > div > figure > figcaption > div
<div class="overflow-hidden" tabindex="0" style="max-width: 55em;"><div class="Space-sc-1ntp1c3-0 SVWDx border-left-width-1" style="border-color: currentcolor;"><p>‘Refugee Astronaut’ in Wellcome’s new permanent gallery ‘Being Human’.</p></div><style>.caption p { display: inline; }</style></div>
 
**Solution:**
Unless an element has a clear role and label, it is advisable that they are not included in focus order. Additionally, only elements with certain roles are generally expected to be focusable, and these are mainly interactive components, such as links or user interface components. In this instance, remove the role of tabindex="0" from the element. Please note this also applies to the text descriptions when clicking the ‘4 images’ button.

**Example:**
<img width="523" alt="Screenshot 2021-05-27 at 11 34 54" src="https://user-images.githubusercontent.com/7315126/119811982-90b99d00-bedf-11eb-98a0-b666f321e58b.png">

**Where**

URL: https://wellcomecollection.org/articles/XYofFREAACQAp-Vl Page: Meet the climate emergency | Wellcome Collection

URL: https://wellcomecollection.org/pages/Wvl00yAAAB8A3y8p Page: Pitch a story | Wellcome Collection

<img width="523" alt="Screenshot 2021-05-27 at 11 33 36" src="https://user-images.githubusercontent.com/7315126/119811782-623bc200-bedf-11eb-9efc-612fa850bf43.png">
